added map (#7)
thesis20/report: sections/Investigation/results.tex

The F1-measure also employed by some of the papers summarizes precision and recall into a single number, defined by \autoref{eqn:f1}.
\begin{equation}
    \label{eqn:f1}
    F1-measure = \frac{2*precision*recall}{precision+recall}
\end{equation}

> Other metrics are derived from these basic information retrieval metrics, with \textit{Mean Average Precision} (MAP) representing a popular metric\cite{ChoosingMetricsEvaluation}.

Reviewer: ```suggestion
Other metrics are derived from these basic information retrieval metrics, with \textit{Mean Average Precision} (MAP) representing a popular metric \cite{ChoosingMetricsEvaluation}.
```

> To calculate MAP, \textit{Average Precision} (AP) must first be calculated.

Reviewer: ```suggestion
To calculate MAP, the first step is to calculate the \textit{Average Precision} (AP).
```

This measure takes each relevant item and calculates precision in relation to the position of the item in the recommendation set.
This is defined in \autoref{eq:averageprecision}, where $P(r)$ is the precision at the $r^{th}$ item and $rel(r)$ is an indicator that the item was relevant, taking the value of either 1 if relevant, or 0 if not.
A point to note about AP is that it rewards recommendation lists with relevant items appearing at the front of the list, leading to a higher AP value in comparison to lists where all relevant items appear nearer the N value. 
\begin{equation}
    \label{eq:averageprecision}
    AP = \frac{\sum\limits_{r=1}^N (P(r) \times rel(r))}{number \: of \: relevant \: documents}
\end{equation}

> Once AP is calculated for a single user $u$, MAP@N for a recommender system is defined through \autoref{eq:map}, where $U$ is the set of all users: 

Reviewer: ```suggestion
Once the AP is calculated for a single user $u$, MAP@N for a recommender system is defined through \autoref{eq:map}, where $U$ is the set of all users: 
```

Reviewer: Skal her ikke være noget med hvad N betyder?
